enable/savage/trait_defs/ui/qt4/svg_editor.py

```diff
@@ -22,7 +22,14 @@
 
 from traitsui.qt4.editor import Editor
 
-from pyface.qt import QtCore, QtSvg
+from pyface.qt import QtCore
+
+# import widget for different package cases
+try:
+    from pyface.qt import QtSvg as QWidget
```

Comment:
I think `QtSvg` is a module here? So shouldn't this be `from pyface.qt.QtSvg import QSvgWidget`?

Reply:
Thanks for the comments. That's good point, then we can import QSvgWidget from both packages without the as renaming. Just adapt this change.